subsys/logging/backends/CMakeLists.txt

```
@@ -15,6 +15,12 @@ zephyr_sources_ifdef(
   log_backend_adsp_mtrace.c
 )
 
+zephyr_sources_ifdef(
+  CONFIG_LOG_BACKEND_BLE
+  log_backend_ble.c
+)
+
+
 zephyr_sources_ifdef(
```

```suggestion
zephyr_sources_ifdef(
  CONFIG_LOG_BACKEND_BLE
  log_backend_ble.c
)

zephyr_sources_ifdef(
```
Small change to be consistent in the file.